tools/build-help-center

```diff
@@ -1,4 +1,34 @@
-#!/usr/bin/env bash
+#!/usr/bin/env python3
 
-./tools/convert-help-center-docs-to-mdx
-(cd ./help-beta/ && /usr/local/bin/corepack pnpm build)
+import os
+import subprocess
+import sys
+
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), ".."))
+
+from scripts.lib.setup_path import setup_path
+
+setup_path()
+
+os.environ["DJANGO_SETTINGS_MODULE"] = "zproject.settings"
+
+import django
+
+django.setup()
+from django.conf import settings
+
+
+def run() -> None:
+    subprocess.check_call(["./tools/convert-help-center-docs-to-mdx"], stderr=subprocess.STDOUT)
```

Comment:
Don’t use `stderr=subprocess.STDOUT`. The default preserves stderr as stderr, and there’s no reason to interfere with that.